buildbots.rst

```diff
@@ -229,7 +229,7 @@ Custom builders
 .. highlight:: console
 
 When working on a platform-specific issue, you may want to test your changes on
-the buildbot fleet rather than just on Travis and AppVeyor.  To do so, you can
+the buildbot fleet rather than just on GitHub Actions and Azure Pipelines.  To do so, you can
```

Comment:
This seems to be the only place in the entire devguide that mentions CI services other than the Python buildbot. Maybe it would be better to remove the outdated line, and later on add more detail on the non-buildbot CI setup?